py_driverstation.py

```diff
@@ -74,15 +74,16 @@ def update(self, new_status):
          to correspond with new status
         """
         # Set status indicator color
-        # Once status is updated, don't update again to avoid unecessary redraws
+        # Once status is updated, don't update again to avoid unecessary
+        #  redraws
         if new_status and not self.status:
             self.status = True
-            self.widget.setStyleSheet(
-                "background-color: %s" % self.status_colors[self.status].name())
+            self.widget.setStyleSheet("background-color: %s" %
+                                      self.status_colors[self.status].name())
```

Comment:
Since we are using Python 3 now, we may want to think about using the `.format()` function instead.

Reply:
Good point, I'll do that.